### DISCUSSION TITLE
OOP-IEC61131-3--Curso-Youtube/en/Clases%20y%20Objetos/Function%20Block/

### DISCUSSION THREAD

**Discussion by giscus** [runtimevic/OOP-IEC61131-3--Curso-Youtube#17] [General]
# OOP-IEC61131-3--Curso-Youtube/en/Clases%20y%20Objetos/Function%20Block/

IEC 61131-3 PLC OOP Course

https://runtimevic.github.io/OOP-IEC61131-3--Curso-Youtube/en/Clases%20y%20Objetos/Function%20Block/

**Comment by Kgbqiang** [comment]
QQ
